CONTRIBUTING.md

```diff
@@ -1,35 +1,44 @@
 # Contributing
 
-Everyone is welcome to contribute to containers. Reach out to team members if you have questions:
+## Requirements
 
-- Matrix chat: [#containers:mozilla.org](https://matrix.to/#/#containers:mozilla.org)
-- Email: containers@mozilla.com
+* Firefox 91.1.0+
+* Git 2.13+
+* Node 7+
```

Comment:
Chore: We should upgrade our node version at some point. We did this for recently for the [Relay add-on](https://github.com/mozilla/fx-private-relay-add-on/blob/main/package.json#L9) and also bumped up web-ext.

Reply:
Agreed. There's a few other deps that could be updated and/or I have no idea if we still need them. I can look at it in a follow-up bug ;)